ovh/resource_me_api_oauth2_client_test.go

```diff
@@ -113,3 +114,42 @@ func TestAccMeApiOauth2Client_configMissingArguments(t *testing.T) {
 		},
 	})
 }
+
+// Test invalid oauth2 clients import IDs
+func TestAccMeApiOauth2Client_importBasic(t *testing.T) {
+	const resourceName = "ovh_me_api_oauth2_client.service_account_1"
+	const okConfigClientCredentials = `
+	resource "ovh_me_api_oauth2_client" "service_account_1" {
+		description = "tf acc test client credentials"
+		name        = "tf acc test client credentials"
+		flow = "CLIENT_CREDENTIALS"
```

Comment:
style: `=` sign could be aligned with the ones above

Reply:
It should be fixed now